packages/driver/src/multidomain/index.js

```diff
@@ -82,7 +82,7 @@ const setup = () => {
 
   specBridgeCommunicator.on('run:domain:fn', (data) => {
     // TODO: await this if it's a promise, or do whatever cy.then does
-    window.eval(`(${data.fn})()`)
+    window.eval(`(${data.fn})`)(data.data)
```

Comment:
Nit: consider destructuring the param (or renaming it)

![https://c.tenor.com/gBHJtNMfw90AAAAM/data-star-trek.gif](https://c.tenor.com/gBHJtNMfw90AAAAM/data-star-trek.gif)